src/neptune_lightgbm/impl/__init__.py

```diff
@@ -71,10 +71,10 @@ class NeptuneCallback:
         for testing without registration.
 
     Args:
-        run (:obj:`neptune.new.run.Run`): Neptune run object.
+        run: Neptune run or handler object.
             A run in Neptune is a representation of all metadata that you log to Neptune.
```

Comment:
Is this how we phrase it now?

Reply:
I think we should call it _namespace handler_, as that's the use case.